src/components/creation/UserGuidedCreation.react.js

```diff
@@ -0,0 +1,27 @@
+// @flow strict
+
+import type {Node} from 'react';
+
+import React from 'react';
+import {makeStyles} from '@material-ui/core/styles';
+import clsx from 'clsx';
+
+function UserGuidedCreation(): Node {
+  function searchAPI(event) {
+    fetch(
+      `https://maps.googleapis.com/maps/api/place/autocomplete/json?input=${event.target.value}key=AIzaSyBf0fe6a03CjZSJBmqKMFHT05KdIBjBpxc`,
```

Comment:
This should not be public. We can use our .env.* file to populate a REACT_APP_* envvar to provide this value.

Reply:
Quieres pelea?

Reply:
Yes, we were just making tests. We forgot to remove the key. Actually, the Places API does not allow us to directly get from their API due to a missing CORS header. I think the way to go is to make the request from a server. Is there a way we can configure a single endpoint, that is independent from the GraphQL API, to handle requests to the Places API?

Reply:
And since the source code is available on the web for the app by inspecting the source, using REACT_APP_* environment variables does not solve the problem of it being public. It's more of a convenient file to store public global variables